cmd/service-controller/policy_controller.go

```diff
@@ -411,7 +411,7 @@ func (c *PolicyController) inferTargetType(target types.ServiceInterfaceTarget)
 		return ""
 	}
 
-	return getBySelector("deployment", "statefulset")
+	return getBySelector("deployment", "statefulset", "deploymentconfig")
```

Comment:
Maybe use the newly introduced `DeploymentObjectType, StatefulsetObjectType and DeploymentConfigObjectType`?

Reply:
Done! (fcf763a48e6f67e501c7d0467388bbb49f6da2b5)